generators/markup/files/render.js

```diff
@@ -0,0 +1,47 @@
+
+// React.
+import { createElement } from 'react';
+import { renderToStaticMarkup, renderToString } from 'react-dom/server';
+
+// Prevent XSS.
+import escape from 'htmlescape';
+
+// Generic page layout.
+import Page from './page';
+import Root from '../';
+
+function body(root, { scripts = [] }) {
+  // TODO: This is where any flux magic would go.
+  return new Promise((resolve) => {
```

Comment:
Dispatch initial action -> Wait for action to resolve -> check for redirect -> Dehydrate state -> Render to string

Some examples I can find:
https://github.com/erikras/react-redux-universal-hot-example/blob/master/src/server.js#L79
https://github.com/tomchentw/redux-universal/blob/master/src/server.js#L45

This is my own implementation of the same pattern from a while ago:
https://github.com/nealgranger/caordafit/blob/master/src/web/server/appMiddleware.js#L38